src/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -208,20 +217,72 @@ public WindowsIdentity(string sUserPrincipalName)
 
```

Comment:
Just above this are two places that can throw; theoretically they should also dispose accessTokenHandle.